src/main/java/com/xiao/users/constants/UserConstants.java

```diff
@@ -0,0 +1,21 @@
+package com.xiao.users.constants;
+
+public final class UserConstants {
+
+    private UserConstants() {
+        // restrict instantiation
+    }
+
+    public static final String SAVINGS = "Savings";
+    public static final String ADDRESS = "123 Main Street, New York";
+    public static final String STATUS_201 = "201";
+    public static final String MESSAGE_201 = "Account created successfully";
+    public static final String STATUS_200 = "200";
+    public static final String MESSAGE_200 = "Request processed successfully";
+    public static final String STATUS_417 = "417";
+    public static final String MESSAGE_417_UPDATE = "Update operation failed. Please try again or contact Dev team";
+    public static final String MESSAGE_417_DELETE = "Delete operation failed. Please try again or contact Dev team";
+    // public static final String  STATUS_500 = "500";
```

Comment:
Maybe just remove the unused status , will add them after when we need them
@nvthiet1995